src/lib/ngsi10/NotifyContextRequest.cpp

```diff
@@ -80,20 +80,27 @@ std::string NotifyContextRequest::toJson
   std::map<std::string, std::string>* replacementsP
 )
 {
-  if ((renderFormat != NGSI_V2_NORMALIZED) && (renderFormat != NGSI_V2_KEYVALUES) && (renderFormat != NGSI_V2_VALUES))
+  if ((renderFormat != NGSI_V2_NORMALIZED) && (renderFormat != NGSI_V2_KEYVALUES) && (renderFormat != NGSI_V2_VALUES) && (renderFormat != NGSI_V2_SIMPLIFIEDKEYVALUES))
   {
     OrionError oe(SccBadRequest, "Invalid notification format");
     alarmMgr.badInput(clientIp, "Invalid notification format");
 
     return oe.toJson();
-  }  
-
-  JsonObjectHelper jh;
-
-  jh.addString("subscriptionId", subscriptionId.get());
-  jh.addRaw("data", contextElementResponseVector.toJson(renderFormat, attrsFilter, blacklist, metadataFilter, replacementsP));
+  }
+  else if ((renderFormat == NGSI_V2_SIMPLIFIEDNORMALIZED) || (renderFormat != NGSI_V2_SIMPLIFIEDKEYVALUES))
```

Comment:
```suggestion
  else if ((renderFormat == NGSI_V2_SIMPLIFIEDNORMALIZED))
```

In general is a variable `x` is equal to `a` it cannot be at the same time equal to `b`, so a caluse `x != b` is not needed.